README.md

```diff
@@ -28,15 +28,15 @@ mount -t debugfs none /sys/kernel/debug
 
 The following kernel configuration is required.
 
-|           Option         |                         Note                         |
-| ------------------------ | ---------------------------------------------------- |
-| CONFIG_DEBUG_INFO_BTF=y  | available since >= 5.3                               |
-| CONFIG_KPROBES=y         |                                                      |
-| CONFIG_PERF_EVENTS=y     |                                                      |
-| CONFIG_BPF=y             |                                                      |
-| CONFIG_BPF_SYSCALL=y     |                                                      |
-| CONFIG_FUNCTION_TRACER=y | /sys/kernel/debug/tracing/available_filter_functions |
-| CONFIG_FPROBE=y          | --backend=kprobe-multi, available since >= 5.18      |
+|           Option         | Backend      |                   Note                               |
+| ------------------------ | ------------------------------------------------------------------- |
```

Comment:
I suspect we need another `|` to make the markdown syntax right: https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-tables#formatting-content-within-your-table

Otherwise, this table is broken (:point_down: 
![image](https://github.com/cilium/pwru/assets/6435624/2b925156-bc84-4ad5-8a3e-f54dec39a5a2)

Reply:
Ooops, fixed in #318